DependencyInjection/NetgenTagsExtension.php

```diff
@@ -45,6 +45,8 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         $loader->load('storage_engines/legacy.yml');
+
+        $loader->load('storage_engines/solr/criterion_visitors.yml');
```

Comment:
This will fail on 2014.11, as `ezpublish.search.common.field_name_resolver` service does not exist.

Moreover, in SPI content type handler, method `getSearchableFieldMap` does not exist. So you will probably need to do some checking before activating this.